### PR TITLE
test: (GX3-4799) Three test cases are added to verify the functionali…

### PR DESCRIPTION
**Pull request by DonnaCas**
…ty of the endpoint buttons

# **Pull Request for branch: test/GX3-4799/tools-qa-elements-buttons**

> "Test(`GX3-4799`): `⚡️ToolsQA | Elements | Buttons`"

## Feature to be Test Automated

- `GX3-4799`: [https://upexgalaxy45.atlassian.net/browse/GX3-4799](url)

## Summary of Changes

> - The functionality is analyzed and the test cases to be executed are selected.
> - Verify the correct operation of the functionality using 3 test cases. 
> - Verify that the repeated execution of the test is successful. 

## Test Results Verification

image

## Git Log

```bash
commit d053d446a208f083b83099d1c64ca18f044899d1 (HEAD -> test/GX3-4799/tools-qa-elements-buttons, origin/test/GX3-4799/tools-qa-elements-buttons)
Author: DonnaCas <donnacas@gmail.com>
Date:   Wed Aug 21 14:40:21 2024 -0500

    test: (GX3-4799) Three test cases are added to verify the functionality of the endpoint buttons
```

## Observations

> N/A

## Checklist

- ✅ My code follows the style guidelines of this project, checking the nomenclature used and correcting any spelling mistakes.

- ✅ I have double-checked my own code and added comments, especially in areas that are difficult to understand.

- ✅New and existing automated tests pass locally, and I have checked that my code works without generating new warnings (IMPORTANT).

- ✅ I have run the Sanity Test pipeline specifically to import my test results into Jira and have verified that my test code works in CI (IMPORTANT).

- ✅ I confirm that I have considered adding new documentation to the project (in the `docs/` folder) if needed. See the description of the PR to see if I added it or not.

### CODE REVIEW

File: cypress/e2e/Tests/Elements/GX3-4799-Buttons.cy.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		cy.get('#doubleClickBtn').dblclick();
+		cy.get('#doubleClickMessage').should('have.text', 'You have done a double click');
+	});
+
+	it('4800 | TC02:  Validate the activation of the "Right Click Me" button and display a message to confirm', () => {
+		cy.get('#rightClickBtn').rightclick();
+		cy.get('#rightClickMessage').should('have.text', 'You have done a right click');
+	});
+	it('4800 | TC03:  Validate the activation of the "Click Me" button and display a message to confirm', () => {
+		cy.get('div.mt-4:nth-child(4) > button').click();
```
Comment (AlexanderWindecker) [inline cypress/e2e/Tests/Elements/GX3-4799-Buttons.cy.js:16]: Bien, me gustaria que atrapes simplifiques este selector button. Y uses "eq" de cypress. Avisame si sale.

**Review by AlexanderWindecker — COMMENTED** [review]
Hola Donna, excelente. Me gustaria que refinamos ese selector para buscar.

File: cypress/e2e/Tests/Elements/GX3-4799-Buttons.cy.js (hunk 1 above)
Comment (DonnaCas) [reply cypress/e2e/Tests/Elements/GX3-4799-Buttons.cy.js:16]: Hola Alexander si tambien funciona utilizando el eq(),  solo no lo utilicé en caso de que cambie algo en la clase, entonces  busqué la fuente de selectores recomendada y segun la docu utilizarlo como selectores hijos era menos probable que mas adelante se cambiara la posicion del botón y test dejara de funcionar.  Pero si se requiere cambiar con gusto lo hago, estaré atenta, gracias por tu recomendación.

Comment (AlexanderWindecker) [reply cypress/e2e/Tests/Elements/GX3-4799-Buttons.cy.js:16]: Bien, si entiendo. Lo decia para que vayamos aplicando eq().

**Review by AlexanderWindecker — APPROVED** [review]

**Review by AlexanderWindecker — APPROVED** [review]

**Pull request merged**